pkg/executor/import_into.go

```diff
@@ -242,6 +270,57 @@ func (*ImportIntoExec) waitTask(ctx context.Context, jobID int64, task *proto.Ta
 	return err
 }
 
+func (e *ImportIntoExec) getSplitRangeFuncs(ctx context.Context) (
+	addTableSplitRange func(), removeTableSplitRangeAndCloseClients func(), err error) {
+	tidbCfg := config.GetGlobalConfig()
+	tls, err := common.NewTLS(
+		tidbCfg.Security.ClusterSSLCA,
+		tidbCfg.Security.ClusterSSLCert,
+		tidbCfg.Security.ClusterSSLKey,
+		net.JoinHostPort("127.0.0.1", strconv.Itoa(int(tidbCfg.Status.StatusPort))),
+		nil, nil, nil,
+	)
+	if err != nil {
+		logutil.Logger(ctx).Warn("fail to NewTLS", zap.Error(err))
+		return nil, nil, err
+	}
+
+	cfg := e.controller.GetLocalBackendCfg(e.Ctx().GetStore().GetKeyspace(), tidbCfg.Path, "")
+
+	//nolint:forcetypeassert
+	store := e.Ctx().GetStore().(tikv.Storage)
+	pdCli := store.GetRegionCache().PDClient()
+	clients, pdCliForTiKV, err := local.CreateClientsForNewBackend(ctx, tls, cfg, pdCli.GetServiceDiscovery())
+	if err != nil {
+		logutil.Logger(ctx).Warn("fail to PrepareClientsForNewBackend", zap.Error(err))
+		return nil, nil, err
+	}
+	intest.Assert(pdCliForTiKV != nil)
+	closeClients := func() {
+		pdCliForTiKV.Close()
+		clients.Close()
+	}
+
+	startKey, endKey := pdCliForTiKV.GetCodec().EncodeRange(
+		tablecodec.EncodeTablePrefix(e.controller.TableInfo.ID),
```

Comment:
It doesn't work for the partition table

Reply:
[`106a23c` (#63467)](https://github.com/pingcap/tidb/pull/63467/commits/106a23ce521796f1ad997c7cee5a002bc1cfc621)